js/sparkswap.js

```diff
@@ -271,7 +271,46 @@ module.exports = class sparkswap extends Exchange {
     }
 
     async fetchOrder (id, symbol = undefined, params = {}) {
-        return this.privateGetV1OrderId ({ id });
+        const order = await this.privateGetV1OrdersId ({ id });
+        let status = 'open';
+        if (order.status === 'CANCELLED') {
+            status = 'cancelled';
+        }
+        if (order.status === 'COMPLETED') {
```

Comment:
All these strings should be part of a constant if we are following that pattern.